backend/spec/requests/api/v1/accounts/project_developers_spec.rb

```diff
@@ -82,7 +82,105 @@
         run_test!
 
         it "returns correct error", generate_swagger_example: true do
-          expect(response_json["errors"][0]["title"]).to eq(I18n.t("errors.messages.user_multiple_accounts"))
+          expect(response_json["errors"][0]["title"]).to eq(I18n.t("errors.messages.user.multiple_accounts"))
+        end
+      end
+    end
+
+    put "Update existing Project Developer" do
+      tags "Project Developers"
+      consumes "multipart/form-data"
+      produces "application/json"
+      security [csrf: [], cookie_auth: []]
+      parameter name: :project_developer_params, in: :formData, schema: {
+        type: :object,
+        properties: {
+          picture: {type: :file},
+          name: {type: :string},
+          about: {type: :string},
+          website: {type: :string},
+          linkedin: {type: :string},
+          facebook: {type: :string},
+          twitter: {type: :string},
+          instagram: {type: :string},
+          project_developer_type: {type: :string, enum: ProjectDeveloperType::TYPES},
+          entity_legal_registration_number: {type: :string},
+          mission: {type: :string},
+          "categories[]": {type: :array, items: {type: :string, enum: Category::TYPES}, collectionFormat: :multi},
+          "impacts[]": {type: :array, items: {type: :string, enum: Impact::TYPES}, collectionFormat: :multi},
+          "location_ids[]": {type: :array, items: {type: :string}, collectionFormat: :multi}
+        }
+      }
+
+      let(:location) { create :location }
+      let(:project_developer) { create :project_developer, :with_locations }
+      let(:user) { create :user }
+      let(:project_developer_params) do
+        {
+          picture: fixture_file_upload("picture.jpg"),
+          name: "Name",
+          about: "About",
+          website: "http://website.com",
+          linkedin: "http://linkedin.com",
+          facebook: "http://facebook.com",
+          twitter: "http://twitter.com",
+          instagram: "http://instagram.com",
```

Comment:
I wanted to say to add language param here and check that it was not updated, but that will land in example, so not good.

Reply:
I will test it at different context, so that should be fine, good point ;)